pkg/proto/nbcontract/v1/kubeletconfig.proto

```diff
@@ -17,14 +17,20 @@ message KubeletConfig {
   // The type of disk to use for the kubelet.
   KubeletDisk kubelet_disk_type = 5;
 
+  // Whether kubelet config should be enabled.
+  bool enable_kubelet_config_file = 6;
```

Comment:
adding to contract because it cannot simply be determined by whether `kubelet_config_file_content` is specified. In AB, kubelet config file content is always populated with some default values, but that doesn't mean we want to use it. In RP, it is determined by a toggle that is defaulted to false. Also depends on other variables such as whether `CustomKubeletConfig` is set.